parts/k8s/windowsnodereset.ps1

```diff
@@ -45,7 +47,13 @@ if ($global:CsiProxyEnabled) {
 $hnsNetwork = Get-HnsNetwork | Where-Object Name -EQ azure
 if ($hnsNetwork) {
     Write-Log "Cleaning up containers"
-    docker ps -q | ForEach-Object { docker rm $_ -f }
+    if ($UseContainerD -eq $true) {
+        ctr -n k8s.io c ls -q | ForEach-Object { ctr -n k8s.io tasks kill $_ }
```

Comment:
might be safer to use ctr.exe here - powershell does weird things when invoking .exe:s

Reply:
and for docker below as well